diff --git a/node_modules/react-native-markdown-editor/lib/src/MarkdownEditor.js b/node_modules/react-native-markdown-editor/lib/src/MarkdownEditor.js
index 349caa2..6780142 100644
--- a/node_modules/react-native-markdown-editor/lib/src/MarkdownEditor.js
+++ b/node_modules/react-native-markdown-editor/lib/src/MarkdownEditor.js
@@ -16,13 +16,16 @@ import { renderFormatButtons } from './renderButtons';
 const FOREGROUND_COLOR = 'rgba(82, 194, 175, 1)';
 const styles = StyleSheet.create({
   composeText: {
-    borderColor: FOREGROUND_COLOR,
     borderWidth: 1,
     flexDirection: 'column',
-    flex: 1,
-    padding: 4,
+    minHeight: 120,
+    padding: 10,
     paddingLeft: 8,
     fontSize: 16,
+    textAlignVertical: 'top',
+    borderRadius: 5,
+    borderColor: '#CCCCCC',
+    backgroundColor: 'white',
   },
   buttonContainer: {
     flex: 0,
@@ -32,13 +35,14 @@ const styles = StyleSheet.create({
     padding: 8,
   },
   preview: {
-    flex: 0.2,
     padding: 5,
     borderWidth: 1,
-    borderColor: FOREGROUND_COLOR,
+    minHeight: 60,
+    borderRadius: 5,
+    borderColor: '#CCCCCC',
+    backgroundColor: 'white',
   },
   screen: {
-    flex: 1,
     flexDirection: 'column',
     alignItems: 'stretch',
     backgroundColor: 'white',
@@ -106,7 +110,7 @@ export default class MarkdownEditor extends React.Component {
       <View style={styles.preview}>
         <ScrollView removeClippedSubviews>
           <MarkdownView styles={markdownStyles}>
-            {this.state.text === '' ? 'Markdown preview here' : this.state.text}
+            {this.state.text === '' ? 'Note Preview Here' : this.state.text}
           </MarkdownView>
         </ScrollView>
       </View>
@@ -126,9 +130,9 @@ export default class MarkdownEditor extends React.Component {
           onChangeText={this.changeText}
           onSelectionChange={this.onSelectionChange}
           value={text}
-          placeholder={'Write a long message'}
+          placeholder={'Note Body'}
           ref={textInput => (this.textInput = textInput)}
-          selection={selection}
+          selection={Platform.OS == 'android' ? selection : null}
         />
         {showPreview ? this.renderPreview() : null}
         <View style={styles.buttonContainer}>
